Makefile

```diff
@@ -158,11 +159,11 @@ rpm: $(TARFILE)
 	rm -r tmp/rpmbuild
 
 ifeq ("$(TEST_SCENARIO)","updates-testing")
-VM_CUSTOMIZE_FLAGS = --run-command 'dnf -y update --setopt=install_weak_deps=False --enablerepo=updates-testing >&2'
+VM_CUSTOMIZE_FLAGS = $(PASTA_HACK) --run-command 'dnf -y update --setopt=install_weak_deps=False --enablerepo=updates-testing >&2'
```

Comment:
Not enjoying the shadowing on this variable.  What's the story there?

Reply:
Mostly that we don't have a `--yes-network` to override the original `--no-network`

Reply:
we can probably do this with an `else` too, though

Reply:
Changed, WDYT?